### PR TITLE
Drop default api url for host of clients

### PR DESCRIPTION
**Pull request by fgebhart**

### CODE REVIEW

File: tests/test_explanation.py [hunk 1]
```diff
@@ -217,7 +210,7 @@ def test_explanation_with_token_granularities_oom_regression(sync_client: Client
 
 ### Response:"""
     target_text = " India won the Battle of Waterloo."
-    model_name = "luminous-base-control"
+    model_name = "luminous-base"
```
Comment (martinreinhardt01) [inline tests/test_explanation.py:213]: Why move away from control?

File: docs/source/index.rst [hunk 2]
```diff
@@ -18,7 +18,7 @@ Synchronous client.
       from aleph_alpha_client import Client, CompletionRequest, Prompt
       import os
 
-      client = Client(token=os.getenv("AA_TOKEN"))
+      client = Client(token=os.getenv("AA_TOKEN"), host="https://inference-api.your-domain.com")
```
Comment (martinreinhardt01) [inline docs/source/index.rst:21]: Why not get the host from the .env, too?

**Review by martinreinhardt01 — COMMENTED** [review]
Easy to grasp MR and nice cleanup on the way. 
Only two very minor questions.

File: tests/test_explanation.py (hunk 1 above)
Comment (fgebhart) [reply tests/test_explanation.py:213]: as it is (currently at least) not available in our new environment

File: docs/source/index.rst (hunk 2 above)
Comment (fgebhart) [reply docs/source/index.rst:21]: good catch. Probably the better approach, will update.

Comment (fgebhart) [reply docs/source/index.rst]: done

File: .github/workflows/integration.yml [hunk 3]
```diff
@@ -24,7 +24,7 @@ jobs:
           poetry run mypy tests --ignore-missing-imports
       - name: Run tests
         run: |
-          poetry run pytest
+          poetry run pytest --color=yes
```
Comment (WieslerAA) [inline .github/workflows/integration.yml:27]: Very important!

**Pull request merged**